lua/core/pack.lua

```diff
@@ -5,6 +5,7 @@ local vim_path = global.vim_path
 local data_dir = global.data_dir
 local lazy_path = data_dir .. "lazy/lazy.nvim"
 local modules_dir = vim_path .. "/lua/modules"
+local user_modules_dir = vim_path .. "/lua/user/modules"
```

Comment:
```suggestion
local user_modules_dir = vim_path .. "/lua/user"
```
It's okay to omit trailing `modules` here (e.g., `.../lua/user/plugins/*.lua` would be enough). The original intention of using `modules` is to follow `Neovim`'s guide on how to write plugins. This can also separate user configs from the default ones to avoid confusion.

Or you may exchange these two for minimum modifications:
```suggestion
local user_modules_dir = vim_path .. "/lua/modules/user"
```

Reply:
Fixed putting user defined plugins in `lua/user/{plugins,configs}`.